tests/Fixer/Phpdoc/PhpdocAddMissingParamAnnotationFixerTest.php

```diff
@@ -329,6 +329,45 @@ function f10(string $foo = "null", $bar) {}',
             ],
             [
                 '<?php
+    /**
+     * Foo
+     * @param null|int $foo
```

Comment:
imho we shouldn't use union if php-level type is not union

```suggestion
     * @param ?int $foo
```

Reply:
@keradus that's syntax less popular and not all ide/tools support it

Reply:
it's always that we have PHP feature first, until all IDEs or SCA tools supports it. yet, we should aim to support new syntax and help to migrate to it, not to keep spreading old ways

Reply:
@keradus Php don't supported PhpDoc. You mix PSR and pseudo-code - it's bad idea:
- loose support by PhpDocumentor(PHP documentator №1), PhpStorm (PHP IDE №1) and other PSR tools
+ undocumented mix supported by... nobody?

Reply:
I think I never saw any `?foo` usage in PHPDocs. And since we now have real union types, I would always favor `foo|null` over `?foo`.

Reply:
@julienfalque [PSR type syntax](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#appendix-a-types)

Reply:
I think when `?int` or `int|null` / `null|int` is already in the param it should not be touched by this fixer, it is about adding not replacing.
If none is there, `null|int` is fine by me.
Than we can have https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3562 to normalize `null|int` into `?int` if wanted by user

Reply:
@SpacePossum `?int` - it's invalid type for psr or phpdocumentator